djstripe/mixins.py

```diff
@@ -38,11 +38,20 @@ class PaymentsContextMixin(object):
 
     def get_context_data(self, **kwargs):
         context = super(PaymentsContextMixin, self).get_context_data(**kwargs)
+        selected_tag = context.get('selected_tag', False)
+        if selected_tag:
+            plan_list = [x for x in djstripe_settings.PLAN_LIST if x['tag'] == selected_tag]
```

Comment:
@aleccool213: This PR needs to be re-worked for the 1.0.0 branch; when that happens, I'm sure this will be resolved.